plugins/autoslab.cpp

```diff
@@ -212,7 +194,7 @@ static void checkslabs(color_ostream &out)
             )
         {
             createSlabJob(ghost);
-            auto fullName = get_first_name(ghost) + " " + get_last_name(ghost);
+            auto fullName = Translation::capitalize(Translation::TranslateName(&ghost->name, false), true);
```

Comment:
I don't think you need `Translation::capitalize()` here, because `addNameWord()` already calls it. If you do find that you need it here, it's worth pointing out that `capitalize(first+last)` isn't equivalent to the previous behavior of `capitalize(first)+capitalize(last)`.

Reply:
I think this would have been equivalent since I passed true to have it capitalize every word, but as you point out, it's a noop and can be removed.

Reply:
Yeah, missed that, I think you're right.